src/pages/People/PersonApprovals.tsx

```diff
@@ -0,0 +1,133 @@
+import dayjs from "dayjs";
```

Comment:
Was this already added to the deps? I assume yes otherwise CI wouldn't pass!

Reply:
Yep, I've be using it for other date handling, the native API isn't great